examples/basicUserStore/user.store.ts

```diff
@@ -0,0 +1,21 @@
+import { Store } from "@leocode/rxstores";
+import { User } from "./user.model";
+
+export class UserStore extends Store<Partial<User>> {
+  constructor() {
+    const initialValue: Partial<User> = {};
+    super(initialValue);
+  }
+
+  async init() {
+    const responseData = await fetch("https://swapi.co/api/people/1/").then(
+      res => res.json()
+    );
+
+    this.value = responseData;
```

Comment:
And if we decide to to use `Loadable`, the value to be emitted will be like `Loadable.ofData(responseData)`.